Source/WebCore/css/CSSCounterStyleRegistry.cpp

```diff
@@ -137,7 +142,7 @@ RefPtr<CSSCounterStyle> CSSCounterStyleRegistry::resolvedCounterStyle(const Atom
 
 CounterStyleMap& CSSCounterStyleRegistry::userAgentCounterStyles()
 {
-    // FIXME: counter-style should get pre-defined counters from UA stylesheet rdar://103021161.
+    // counter-style should get pre-defined counters from UA stylesheet, but we are inserting decimal manually here because it is the last fallback.
     auto initialStyles = []() {
         CounterStyleMap map;
         map.add("decimal"_s, CSSCounterStyle::createCounterStyleDecimal());
```

Comment:
I'd prefer that we remove this in favor of the new mechanism, and that we assert that the decimal counter style exists in the places we need it.